### PR TITLE
Tailwind text

### LINKED ISSUE #9192: replace `is-size-` with tailwind equivalent
ref #9013

### PR DESCRIPTION
**Pull request by pbkompasz**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [X] Refactoring

## Context

- [X] Closes #9192
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [X] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt&usdamount=50&donation=true)

#### Community participation

- [ ] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/base/BaseCollectionForm.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <h2 class="title is-size-3">
+    <h2 class="title text-3-5xl">
```
Comment (roiLeo) [inline components/base/BaseCollectionForm.vue:3]: typo?

**Review by roiLeo — CHANGES_REQUESTED** [review]
typo on all `text-3-5xl` should be `text-5xl`

File: components/base/BaseCollectionForm.vue (hunk 1 above)
Comment (pbkompasz) [reply components/base/BaseCollectionForm.vue:3]: is-size-3 is 2rem and tailwind has text-3xl(font-size: 1.875rem) and text-4xl(font-size: 2.25rem). I opted to keep the original size of 2rem. Should I change it to 1.875rem?

Comment (roiLeo) [reply components/base/BaseCollectionForm.vue:3]: - https://tailwindcss.com/docs/font-size#customizing-your-theme
or 
- https://tailwindcss.com/docs/font-size#arbitrary-values

choose one of the solutions

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

**Comment by prury** [comment]
https://github.com/kodadot/nft-gallery/assets/36627808/00d9c004-59b2-459f-a0d4-0e52d6483ff9

https://github.com/kodadot/nft-gallery/assets/36627808/f5fc083c-5ad6-4470-a99c-766869cdf003

**Comment by prury** [comment]
last thing:

https://github.com/kodadot/nft-gallery/assets/36627808/a56d4054-c961-4366-903f-e20f6b898165

line spacing changed too much

**Review by Jarsen136 — APPROVED** [review]
code conflict please

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 6.69 USD/DOT ~ 7.474 $DOT
🧗 [16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt](https://polkadot.subscan.io/account/16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt)
🔗 [0xb6854ae255fa95ec0e3b62db2193a8ad1b266be71c878df33ed084a8a0dc913b](https://polkadot.subscan.io/extrinsic/0xb6854ae255fa95ec0e3b62db2193a8ad1b266be71c878df33ed084a8a0dc913b)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**